<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>495</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>580</width>
    <height>490</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>tex equation</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="myPlainTextEdit" name="plainTextEditEq">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>441</width>
      <height>261</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="overwriteMode">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonConvert">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>250</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>convert</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditName">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>20</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>file-name</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxMagnitude">
    <property name="geometry">
     <rect>
      <x>511</x>
      <y>50</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>2</number>
    </property>
    <property name="maximum">
     <number>20</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>50</y>
      <width>50</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>size</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxFont">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>100</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>default</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>arev</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>fourier</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>80</y>
      <width>50</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>font</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxImageType">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>150</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>png</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>jpg</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>tif</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>130</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>image type</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsViewPreview">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>560</width>
      <height>130</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>50</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>preview</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonPreview">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>210</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>preview</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxAutoPreview">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>190</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>auto preview</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>580</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>file</string>
    </property>
    <addaction name="actionLoad"/>
    <addaction name="actionImport_2"/>
   </widget>
   <widget class="QMenu" name="menuPath">
    <property name="title">
     <string>path</string>
    </property>
    <addaction name="actionPlatex"/>
    <addaction name="actionDvips"/>
    <addaction name="actionImageMagick"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuPath"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionLoad">
   <property name="text">
    <string>load</string>
   </property>
  </action>
  <action name="actionPlatex">
   <property name="text">
    <string>platex</string>
   </property>
   <property name="toolTip">
    <string>platex</string>
   </property>
  </action>
  <action name="actionImageMagick">
   <property name="text">
    <string>ImageMagick</string>
   </property>
  </action>
  <action name="actionDvips">
   <property name="text">
    <string>dvips</string>
   </property>
  </action>
  <action name="actionImport_2">
   <property name="text">
    <string>import</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>myPlainTextEdit</class>
   <extends>QPlainTextEdit</extends>
   <header>myplaintextedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
